### PR TITLE
[DowngradePhp80] Fix Type boolean and array on DowngradeNamedArgumentRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
Should be fixed now 
Screen Shot 2021-06-23 at 23 42 00

**Comment by samsonasik** [comment]
@TomasVotruba Finally fixed 🎉 

Screen Shot 2021-06-24 at 00 05 27


It actually need improvement as [previously](https://github.com/rectorphp/rector/blob/0ef9715e0d698e7e34292192656b72e0bc162893/src/Configuration/ConfigurationFactory.php#L23), it precisely got : 

```php
return new \Rector\Core\ValueObject\Configuration(\false, \true, \false, \Rector\ChangesReporting\Output\ConsoleOutputFormatter::NAME, $fileExtensions, [], \true);
```

and [now](https://github.com/rectorphp/rector/blob/3419d090b1385998a13a50f70c0391bc2069c5cd/src/Configuration/ConfigurationFactory.php#L23), it got: 

```php
return new \Rector\Core\ValueObject\Configuration(\false, \true, \false, 'console', $fileExtensions, array(), \true);        

```

**Comment by TomasVotruba** [comment]
Thank you, that's good news 🤗

File: rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php [hunk 1]
```diff
@@ -217,10 +214,6 @@ private function mapPHPStanTypeToExpr(?Type $type): ?Expr
             return null;
         }
 
-        if ($type instanceof NullType) {
-            return new ConstFetch(new Name('null'));
-        }
-
-        throw new NotImplementedYetException();
+        return new ConstFetch(new Name($type->describe(VerbosityLevel::value())));
```
Comment (TomasVotruba) [inline rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php:217]: This might fail for string and number

**Pull request merged**